src/Gaufrette/Adapter/Ftp.php

```diff
@@ -366,12 +378,31 @@ private function isDir($directory)
             return true;
         }
 
-        if (!@ftp_chdir($this->getConnection(), $directory)) {
-            return false;
+        $chDirResult = false;
```

Comment:
This line can be removed ;)
The scope of this var should be in the `try` block only.

Reply:
Correct.